src/objects.rs

```diff
@@ -87,17 +87,29 @@ macro_rules! impl_codes {
     };
 }
 
+pub type PropertiesMap = std::collections::BTreeMap<String, String>;
+/// Helper to create a map of properties. Take a list of tuples `(key, value)`.
+#[macro_export]
+macro_rules! properties_map {
+    ($(($k:expr, $v:expr)),*) => {{
+        let mut map = std::collections::BTreeMap::default();
+            $(
+                map.insert($k, $v);
```

Comment:
So this is blindly updating if there is a conflict (last is kept).
I don't see any problem in this PR about that init.
Just commenting to ask if anyone anticipates behavior change on that point.

Reply:
I believe [this comment](https://github.com/CanalTP/transit_model/pull/763#issuecomment-829297250) answer your comment.